hackathon_site/dashboard/frontend/src/App.scss

```diff
@@ -21,3 +21,39 @@
         white-space: nowrap;
     }
 }
+
+// for OrdersTable
+.order-Submitted-icon {
```

Comment:
added these styles to App.scss so i can use the className in the form a string, is it ok to add it here?
using the OrdersTable.module.scss has different syntax so it doesn't seem like I can use the className as a string

Reply:
I wouldn't recommend it since this file is specifically for App. Ideally you would use OrdersTable.module.scss